apps/staking/src/app/api/publishers-ranking/route.tsx

```diff
@@ -0,0 +1,10 @@
+import { NextResponse } from "next/server";
+
+export async function GET() {
+  const publisherRankingsResponse = await fetch(
+    "https://www.pyth.network/api/publishers-ranking?cluster=pythnet",
+  );
+
+  const publisherRankings = (await publisherRankingsResponse.json()) as JSON;
+  return NextResponse.json(publisherRankings);
+}
```

Comment:
In this case, since the endpoint is going to be a straight up proxy, you can actually avoid having the endpoint entirely by using [nextjs rewrites](https://nextjs.org/docs/app/api-reference/next-config-js/rewrites#rewriting-to-an-external-url)

Reply:
done.